membership.md

```diff
@@ -0,0 +1,90 @@
+Membership and Commit Rights
+============================
+
+This document covers processes to be granted membership in the Dask
+GitHub organization and commit rights to Dask repositories.
+
+
+Owners
+------
+
+There is a set of individuals who have ownership privileges over the entire
+GitHub organization.  They have commit rights to all repositories by default
+and are involved in creating and removing members.  These individuals are
+listed at https://github.com/orgs/dask/people .
+
+We ask that owners ensure that their membership is public.
+
+New owners are made by lazy consensus of previous owners.  All owners will be
+made aware of a new candidate.  Any -1 vote bars a candidate.  We ask that a
+majority of owners respond for quorum, and that those votes come from people
+employed/associated to at least three institutions.
+
+
+Members
+-------
+
+New members can be made by any owner if the following criteria are met:
+
+1.  The owner thinks it's a good idea.
+2.  Some other owner employed by a different institution thinks it's a good
+    idea.
+3.  The new member has supported the project several times, either through code
+    or otherwise, and has demonstrated a willingness to help solve other
+    people's problems, not just their own.
+4.  The new member demonstrates some degree of familiarity of open source
+    practices and online courtesy.
+
+Members can be given write privileges to any repository by an owner by the
```

Comment:
Does this mean that a contributor has to be a Dask org member before giving him/her write priviledges?

Reply:
I did not intend to imply that.  We do have cases today where people have write priveleges without being members of the org.  Often this happens when we bring in an external repository.  

I'll change the language in a bit.